packages/react-native/ReactCommon/react/renderer/mounting/ShadowTree.cpp

```diff
@@ -415,6 +441,10 @@ CommitStatus ShadowTree::tryCommit(
       (commitOptions.shouldYield && commitOptions.shouldYield())) {
     return CommitStatus::Cancelled;
   }
+        
+  if (oldRootShadowNode->getConcreteProps().layoutContext.fontSizeMultiplier != newRootShadowNode->getConcreteProps().layoutContext.fontSizeMultiplier) {
+      newRootShadowNode = dirtyTree(newRootShadowNode);
+  }
```

Comment:
I'm not sure about adding this explicit check inside the commit logic. 

Maybe it should be a commit hook?